README.md

```diff
@@ -87,6 +87,20 @@ Read more about [how KRR works](#how-it-works) and [KRR vs Kubernetes VPA](#diff
 
 ## Installation
 
+### Requirements
+
```

Comment:
I think you can remove `running either on your cluster or centralized.`

We use the term "centralized" a lot, but not sure its obvious to the user what we mean.